### DISCUSSION TITLE
Support for Arbitrum Goerli

### DISCUSSION THREAD

**Discussion by frankfka** [WalletConnect/walletconnect-monorepo#2756] [Developer Support]
Hey all,

Is WalletConnect V2 planning to support Arbitrum Goerli anytime soon? In the example [react app](https://react-app.walletconnect.com/), `Arbitrum Rinkeby` will display available wallets. I'm using `wagmi` and passing `Arbitrum Goerli` as a chain doesn't populate any app connection options

**Comment by alexeyl2014** [comment]
i dont know
